runtime/docker/auth.go

```diff
@@ -62,7 +62,7 @@ func GetDockerConfig(configPath string) (*DockerConfig, error) {
 
 	file, err := os.ReadFile(dockerConfigPath)
 	if err != nil {
-		log.Infof("Could not read docker config: %v", err)
+		log.Debugf("Could not read docker config: %v", err)
```

Comment:
I think it would be cool to check if error is FileNotFound and do a debug logging
any other error (like poorly crafted config) should be info'ed as this might be require users attention